src/sqlancer/databend/DatabendErrors.java

```diff
@@ -30,7 +30,7 @@ public static void addExpressionErrors(ExpectedErrors errors) {
         if (DatabendBugs.bug9264) {
             errors.add("assertion failed: offset + length <= self.length");
         }
-        errors.add("validity's length must be equal"); // bug
+        // errors.add("validity's length must be equal"); // bug
```

Comment:
Perhaps we can remove this comment instead?

Reply:
This error was triggered with a small probability before, I forgot to leave a log.

Reply:
Okay, I see. Then let's leave the comment in for now.